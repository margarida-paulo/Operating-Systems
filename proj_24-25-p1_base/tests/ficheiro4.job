WRITE [(Ana,Martins)(Ricardo,Oliveira)]
WRITE [(João,Pinto)(Maria,Costa)]
WRITE [(Pedro,Santos)(Marta,Ribeiro)]
WRITE [(Clara,Sousa)(Luís,Carvalho)]
WRITE [(André,Monteiro)(Inês,Mendes)]
WRITE [(Rui,Nogueira)(Carla,Lopes)]
WRITE [(Tiago,Pereira)(Diana,Rocha)]
WRITE [(Gonçalo,Cruz)(Patrícia,Silva)]
WRITE [(Alexandra,Barros)(Fernando,Teixeira)]
WRITE [(Mariana,Amorim)(José,Martins)]
WRITE [(Francisco,Ferreira)(Laura,Gomes)]
WRITE [(Sofia,Monteiro)(Miguel,Pires)]
WRITE [(Raquel,Moura)(Hugo,Vieira)]
WRITE [(Carlos,Ramos)(Sara,Marques)]
WRITE [(Joana,Almeida)(Rafael,Antunes)]
WRITE [(Beatriz,Moreira)(Rodrigo,Azevedo)]
WRITE [(Tomás,Barbosa)(Carolina,Bastos)]
WRITE [(Leonor,Ferreira)(Henrique,Coelho)]
WRITE [(Daniela,Lima)(Bruno,Neves)]
WRITE [(Fábio,Matos)(Catarina,Castro)]

SHOW
SHOW
SHOW
