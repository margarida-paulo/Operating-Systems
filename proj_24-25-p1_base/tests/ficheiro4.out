Invalid command. See HELP for usage
